.github/workflows/update-zarf-injector.yaml

```diff
@@ -0,0 +1,59 @@
+name: Zarf Injector Update
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  update-injector-version:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v4
+
+      - name: Fetch zarf-config.toml from zarf-dev
+        run: |
+          git clone https://github.com/zarf-dev/zarf.git
```

Comment:
We should `git checkout ${zarf_version}` before running `cp` on the file 

zarf_version=[this](https://github.com/defenseunicorns/delivery-zarf-init/blob/main/zarf-config.yaml#L9) field)?

Reply:
I think it's resolved now?